scout/server/blueprints/variants/controllers.py

```diff
@@ -1079,10 +1083,7 @@ def check_form_gene_symbols(
             isinstance(hgnc_genes, list) is False
         ):  # Gene was not found using provided symbol, aliases were returned
             hgnc_genes = list(hgnc_genes)
-            flash(
-                f"Could not find a gene with symbol '{hgnc_symbol}'. Search was extended to genes using symbol as an alias.",
-                "warning",
-            )
+            outdated_symbols.add(hgnc_symbol)
```

Comment:
This is already done a few lines down no? In that case could also be the gene panel that contains an outdates symbol.. Do we want to add another key/flash message value to the warnings dictionary to distinguish the 2 cases?

Reply:
I was considering that: it would be clearers - the only issue is that it would be longer if there are both.

Reply:
Maybe its good. Then we can sharpen the warning messages a bit as well - they are intentionally vague as is. 😁